README.rst

```diff
@@ -21,23 +23,98 @@ pythx
         :target: https://coveralls.io/github/dmuhs/pythx?branch=master
 
 
+PythX is a library for the Mythx_ smart contract security analysis platform.
+
+.. code-block::
+
+        Usage: pythx [OPTIONS] COMMAND [ARGS]...
+
+        Options:
+        --help  Show this message and exit.
+
+        Commands:
+        check    Submit a new analysis job based on source code, byte code, or...
```

Comment:
This is more of an UX thing but could we rename `check` to `verify`?
Check is a bit ambiguous in its meaning, since it can refer to validation as well as status. ie: you can '_check if the contract is valid_' BUT you can also '_check the analysis status_'
It would also be nice to match with truffle-security for when developers are coming from there.

Reply:
I agree that `check` is suboptimal. I have also thought abut `verify`, but wanted to avoid confusion with formal verification. Technically we don't verify or validate anything but rather analyze. So my proposal for this would be `submit` or `analyze`. WDYT?

Reply:
That's a good point about `verify`. Both `submit` and `analyze` are really good options. I'm leaning more towards `analyze` since I think it better conveys what is happening to the contracts once they are submitted.

Reply:
I'll create an issue for that as it's a larger refactoring :+1: `analyze` sounds fine